### PR TITLE
fix: Icon updated for "Ticket Buyer Information"

### LINKED ISSUE #5624: Order Page and Complete Order Page: Choose better icon for "Ticket Buyer Information"
The ticket buyer information uses the same icon as the ticket holder's information. Please find and implement a better suited different icon, maybe something related to money or payment?

Screenshot from 2020-11-13 22-29-04

### PR DESCRIPTION
**Pull request by MeghalBisht**
Fixes #5624 

#### Short description of what this resolves:
Icon changed for the "Ticket Buyer" information. 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

Attachments -

### CODE REVIEW

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
Screenshot_20201114-181302
 Add this icon on order form also.

File: app/templates/orders/pending.hbs [hunk 1]
```diff
@@ -20,64 +20,58 @@
...
+        <div>
+          {{#if this.isStripe}}
+            <StripeCheckout
+              @image="https://stripe.com/img/documentation/checkout/marketplace.png"
+              @currency={{this.model.order.event.paymentCurrency}}
+              @locale="auto"
+              @name="Open Event"
+              @class="ui right labeled icon blue button"
+              @description={{this.paymentDescription}}
+              @amount={{this.paymentAmount}}
```
Comment (snitin315) [inline app/templates/orders/pending.hbs:42]: Please revert these changes.

**Review by snitin315 — CHANGES_REQUESTED** [review]
Need to rebase!

**Review by Saumyachitransh01 — COMMENTED** [review]
Use "money bill alternate outline icon".
This icon might look better.

**Comment by MeghalBisht** [comment]
> Use "money bill alternate outline icon".
> It might look better.

It is being used in #5646

**Comment by Raunaksingh100** [comment]
> > Use "money bill alternate outline icon".
> > It might look better.
> 
> It is being used in #5646 

Ask the mentors for any suggestions

**Comment by Raunaksingh100** [comment]
> > Use "money bill alternate outline icon".
> > It might look better.
> 
> It is being used in #5646 

You can also use it doesn't matter 
The purpose of  your pr and my pr is different

**Comment by Saumyachitransh01** [comment]
> It is being used in #5646

Watch carefully, it's not the same

**Review by mariobehling — APPROVED** [review]
Looks good to me. Thank you!

**Pull request merged**